### DISCUSSION TITLE
Meven Korgal and others APE scripts

### DISCUSSION THREAD

**Discussion by cressidagp** [arcemu/arcemu#448] [Show and tell]
Hi there community!

I did a basic script for Meven Korgal, probably not good enough for a PR but it will do the job. 

TODO: add a check to avoid chat if npc is in combat.

-- **Meven Korgal (random chat)**
https://pastebin.com/FtyXRAaN

-- **Knight Defender Zunade (gossip script)**
https://pastebin.com/NGYBtHH3

-- **Sputtervalve (gossip script)**
https://pastebin.com/j6z23fw2

-- **Jenal (gossip script)**
https://pastebin.com/S88tfJrd

-- **Fandral Staghelm (gossip script)**
https://pastebin.com/guPwFj1C

edit: added more APE scripts.

**Comment by cressidagp** [comment]
-- Junior Apothecary Holland (chat on random waypoint)
https://pastebin.com/kwCeBmApl

-- Devlin Agamand (random chat)
https://pastebin.com/hjpdGDE2

**Comment by cressidagp** [comment]
-- Ol Emma (chat on random waypoint)
https://pastebin.com/AtxcbTkk

-- Elders (gossip script)
https://pastebin.com/92vex7ve

-- Arcanist Vandrin (gossip script)
https://pastebin.com/EW8NWzza

**Comment by cressidagp** [comment]
-- Khadgar (gossip script) [ i gues this one can be improved ]
https://pastebin.com/D8tCL3Ab

**Comment by dfighter1985** [comment]
Wow! Thanks for all of your work!

**Reply by cressidagp** [reply]
Just a little grain of sand... 😊

**Comment by cressidagp** [comment]
It was more hard than i expected but finally my first script with the new paradigm its done. Also added both new classes to community wiki.

-- **Supervisor Raelen**
Paradigm: OO
Description: random chat with emote when a peasant its on a short distance, then disable for 20 secs.
https://pastebin.com/vv5JGXjP
